.github/workflows/health.yaml

```diff
@@ -8,7 +8,9 @@ jobs:
   health:
     uses: dart-lang/ecosystem/.github/workflows/health.yaml@main
     with:
-      ignore_license: "**.g.dart"
       sdk: dev
+      channel: dev
```

Comment:
Both `sdk` and `channel` are meaningful? What is the difference?

Reply:
`sdk` is for the Dart version, `channel` is for the Flutter version.